core/templates/dev/head/pages/exploration-editor-page/improvements-tab/feedback-improvement-task/feedback-improvement-task.directive.ts

```diff
@@ -62,8 +62,21 @@ angular.module('oppia').directive('feedbackImprovementTask', [
           };
 
           $scope.getLocaleAbbreviatedDatetimeString = function() {
```

Comment:
This code change is not clean because the function does not return what it says on the tin (the return value is not a "locale abbreviated datetime string").

I think a cleaner approach would be to do something like the following in html:

```
<span ng-if="wasUpdatedAtMost24HoursAgo()">at</span>
<span ng-if="!wasUpdatedAtMost24HoursAgo()">on</span>
<[getLocaleAbbreviatedDatetimeString()]>
```

Reply:
Yeah, it should ideally return string.In the new commit, I was thinking to save the date string in scope and then check whether `wasUpdatedAtMost24HoursAgo` to avoid computation of `getLocaleAbbreviatedDatetimeString()` twice. Though I'm not sure its a good practice or not.